src/convertToObject.js

```diff
@@ -11,8 +11,24 @@
  *
  * @return {object}
  */
+/* eslint-enable */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const styleToObject = {};
```

Comment:
A variable must report what it stores:
```suggestion
  const convertedStyles = {};
```